### ISSUE TITLE
Installing locall and following instructions exactly with virtual envirnoment

### ISSUE THREAD

**Issue by anumerico** [NASA-IMPACT/hls-foundation-os#28]
Still getting errors, thats the current one:

RuntimeError: TemporalEncoderDecoder: TemporalViTEncoder: Attempting to deserialize object on a CUDA device but torch.cuda.is_available() is False. If you are running on a CPU-only machine, please use torch.load with map_location=torch.device('cpu') to map your storages to the CPU.

**Comment by CarlosGomes98** [comment]
Indeed, for running on CPU I merged a new PR that should fix this. You should be able to pull again from main and this should be fixed.

**Comment by anumerico** [comment]
Im installing NVIDIA drivers, CUDA and so on to see if it fixes the issue..

**Comment by anumerico** [comment]
Cool I saw what you did @CarlosGomes98 , I did git pull and hopefully it works with non CUDA installations, now the issue is I managed to get CUDA but is an old GPU that is not supported:
Found GPU0 NVIDIA GeForce **** which is of cuda capability 3.5.
    PyTorch no longer supports this GPU because it is too old.
    The minimum cuda capability supported by this library is 3.7.

**Comment by anumerico** [comment]
That sorted the issue:
https://stackoverflow.com/questions/53266350/how-to-tell-pytorch-to-not-use-the-gpu
